host/bcm283x/bcm283xsmoketest/bcm283xsmoketest.go

```diff
@@ -80,9 +87,9 @@ func (s *SmokeTest) waitForEdge(p gpio.PinIO) <-chan bool {
 	return c
 }
 
-// testClock tests .PWM() for a clock pin.
```

Comment:
Keep this test, add testPWMbyDMA as a separate test.

Reply:
Reverted but please note I actually disabled clock driven PWM for all pins.

Reply:
I removed testClock() again because it does not make sense now. (ex. expects error for arbitrary duty)
If you want to keep clock driven PWM for pins like GPIO6,  please let me know.